.github/workflows/_test.yml

```
@@ -13,6 +13,7 @@ jobs:
       matrix:
         node: ["20", "lts/*", "current"]
     permissions: {}
+    continue-on-error: ${{ matrix.node != "current" }}
```

actionlint (1.7.4)

got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)